Update SEO (#115)
OpenLiberty/guide-microprofile-fallback: README.adoc

:page-tags: ['MicroProfile']
:page-permalink: /guides/{projectid}
:page-related-guides: ['rest-intro', 'cdi-intro', 'microprofile-config', 'circuit-breaker']

> :page-seo-title: Building fault-tolerant microservices with Eclipse MicroProfile Fault Tolerance's Fallback policy

Reviewer: ```suggestion
:page-seo-title: Building fault-tolerant Java microservices with Eclipse MicroProfile Fault Tolerance's fallback policy
```

> :page-seo-description: A tutorial on how to build resilient Java microservices that deal with failed executions and monitor the fault tolerance methods using MicroProfile Fault Tolerance and MicroProfile Metrics.

Reviewer: ```suggestion
:page-seo-description: A tutorial and an example on how to build resilient Java microservices to handle failed executions using MicroProfile Fault Tolerance and monitor fault-tolerant methods using MicroProfile Metrics.
```
